packages/manager/apps/hpc-vmware-managed-vcd/src/utils/refetchConditions.ts

```diff
@@ -0,0 +1,12 @@
+import IVcdDatacentre from '@/types/vcd-datacenter.interface';
+import IVcdOrganization from '@/types/vcd-organization.interface';
+
+export type UpdatableResource = IVcdOrganization | IVcdDatacentre;
+
+const targetSpecKey = 'configure-target-spec';
+
+export const isUpdatingTargetSpec = (resource: UpdatableResource) =>
+  resource?.currentTasks?.some((task) => task.type === targetSpecKey);
```

Comment:
```suggestion
export const isUpdatingTargetSpec = (resource?: UpdatableResource) =>
  resource?.currentTasks?.some((task) => task.type === targetSpecKey);
```

Vu que tu test si resource est définie tu peux acceté les `undefined`